.idea/compiler.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
those files are from intellij idea I suppose, they should not be included in commit, it will cause conflict for all other users using that IDE, you can do a `push -f` to have a clean commit w/o `.idea` included.

Reply:
Thanks for useful advice! I'll do it soon!

Reply:
Hello! I'd want to ask you how I'll can make new commit without files .idea included. 
Will I have to clean files with extending .idea from my branch at first before I can make new commit with 'push -f'? Am I understanding your comment correctly? Thanks for your answer and sorry for my imperfect English!